portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -15,9 +15,10 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    DatastoreService datastore = DataServlet.datastore;
-    List<Key> keys = DataServlet.keys;
-    datastore.delete(keys);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment").setKeysOnly();
```

Comment:
Just a note that we'll probably want to find a common place to keep these constants (like "Comment") so that if they need to change, they will only need to be changed in one spot. No need to do anything now - just something to think about...

Reply:
Yea that could be a static class member later.